src/components/UiLibrary.js

```diff
@@ -182,22 +201,63 @@ const UiLibrary = ( ) => {
             />
           ) )}
         </View>
-        <Heading2 className="my-2">InlineUser</Heading2>
-        <Body2 className="my-2">InlineUser component</Body2>
-        <InlineUser
-          user={currentUser || {
-            icon_url:
-            "https://static.inaturalist.org/attachments/users/icons/1044550/medium.jpg?1653532155",
-            login: "turtletamer74"
-          }}
+
+        <Heading2 className="my-2">User Icons</Heading2>
+        <View className="flex flex-row justify-between">
+          <View>
+            <Body2 className="my-2">UserIcon</Body2>
+            <UserIcon
+              uri={{
+                uri: "https://static.inaturalist.org/attachments/users/icons/1044550/medium.jpg?1653532155"
+              }}
+            />
+          </View>
+          <View>
+            <Body2 className="my-2">InlineUser</Body2>
+            <InlineUser
+              user={
+                currentUser || {
+                  icon_url:
+                    "https://static.inaturalist.org/attachments/users/icons/1044550/medium.jpg?1653532155",
+                  login: "turtletamer74"
+                }
+              }
+            />
+          </View>
+          <View>
+            <Body2 className="my-2">
+              InlineUser for a user that has no icon set
+            </Body2>
+            <InlineUser user={{ login: "frogfinder23" }} />
+          </View>
+        </View>
+
+        <Heading2>Tabs component</Heading2>
+        <Tabs
+          tabs={[
+            {
+              id: "TAB1",
+              text: "Tab1",
+              onPress: () => {
+                console.log( "Tab1" );
+              }
+            },
+            {
+              id: "TAB2",
+              text: "Tab2",
+              onPress: () => {
+                console.log( "Tab2" );
+              }
+            }
+          ]}
+          activeId="TAB1"
         />
-        <Body2 className="my-2">InlineUser component for a user that has no icon set</Body2>
-        <InlineUser user={{ login: "frogfinder23" }} />
 
         <Heading2 className="my-2">Quality Grade Status</Heading2>
         <View className="flex flex-row justify-between">
           <View>
             <Body2 className="text-center">Research</Body2>
+            { /* TODO: refactor to not have color prop because we only need black and white */}
```

Comment:
It's also strange to me here that we're using colors that are not part of the tailwind theme. I would think every color we use, even black, should be accessed as a variable. But not really part of this PR.